### ISSUE TITLE
Error handling for the non-programmer is not intuitive

### ISSUE THREAD

**Issue by Bachibouzouk** [RAMP-project/RAMP#45]
Feedback from real usecases could help us prioritize here

**Comment by FLomb** [comment]
Are you suggesting implementing explicit warnings? I guess this goes hand-in-hand with #48 , doesn't it?

**Comment by Bachibouzouk** [comment]
Yes. Although the tests are meant to help the developers to see everything is alright and the error-handling is there for the users to understand what is wrong in their input while using the tool.
